sonar-helm-for-iac/main_test.go

```diff
@@ -222,13 +223,13 @@ metadata:
 spec: foo
 `
 
-	result, _ := evaluateTemplate(converters.NewTemplateSources("templates/a.yaml", converters.Files{
+	result := evaluateTemplate(converters.NewTemplateSources("templates/a.yaml", converters.Files{
 		"templates/a.yaml":     template,
 		"templates/empty.yaml": make([]byte, 0),
 		"values.yaml":          values,
 		"Chart.yaml":           DefaultChartYaml}))
 
-	assert.Equal(t, expected, result)
+	assert.Equal(t, expected, result.Template)
```

Comment:
Please add an assertion for `result.error` (and in next tests if it makes sense)